shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/test/java/org/apache/shardingsphere/encrypt/rewrite/token/generator/EncryptGeneratorBaseTest.java

```diff
@@ -76,6 +89,40 @@ protected static InsertStatement createInsertStatement() {
         return result;
     }
     
+    /**
+     * SQL: UPDATE t_user SET pwd = '654321' WHERE name = 'LiLei' AND pwd = '123456'.
+     */
+    protected static UpdateStatementContext createUpdatesStatementContext() {
+        MySQLUpdateStatement mySQLUpdateStatement = new MySQLUpdateStatement();
+        mySQLUpdateStatement.setTableSegment(createTableSegment(TABLE_NAME));
+        mySQLUpdateStatement.setWhere(createWhereSegment());
+        mySQLUpdateStatement.setSetAssignment(createSetAssignmentSegment());
+        return new UpdateStatementContext(mySQLUpdateStatement);
+    }
+    
+    /**
+     * SQL-template: WHERE name = 'LiLei' AND pwd = '123456'.
+     */
+    private static WhereSegment createWhereSegment() {
+        BinaryOperationExpression nameExpression = new BinaryOperationExpression(10, 24,
+                new ColumnSegment(10, 13, new IdentifierValue("name")), new LiteralExpressionSegment(18, 22, "LiLei"), "=", "name = 'LiLei'");
+        BinaryOperationExpression pwdExpression = new BinaryOperationExpression(30, 44,
+                new ColumnSegment(30, 32, new IdentifierValue("pwd")), new LiteralExpressionSegment(40, 45, "123456"), "=", "pwd = '123456'");
+        return new WhereSegment(0, 0, new BinaryOperationExpression(0, 0, nameExpression, pwdExpression, "AND", "name = 'LiLei' AND pwd = '123456'"));
+    }
+    
+    /**
+     * SQL-template: SET pwd = '654321'.
```

Comment:
Please remove useless javadoc in unit test.

Reply:
ok